src/components/NavBar/NavBar.jsx

```diff
@@ -1,36 +1,55 @@
 import React, { useState, useEffect } from 'react';
-import PropTypes from 'prop-types';
-import { NavLink, useNavigate } from 'react-router-dom';
-import { ChakraProvider, Button, useDisclosure } from '@chakra-ui/react';
+import PropTypes, { instanceOf } from 'prop-types';
+import { NavLink, useNavigate, useLocation } from 'react-router-dom';
+import {
+  ChakraProvider,
+  useDisclosure,
+  Link,
+  Box,
+  Flex,
+  Spacer,
+  Modal,
+  ModalOverlay,
+  ModalContent,
+  IconButton,
+  ModalBody,
+  ModalCloseButton,
+} from '@chakra-ui/react';
 import UploadCSV from '../UploadCSV/UploadCSV';
-
 import styles from './NavBar.module.css';
 import FYALogo from '../../assets/fya-logo.png';
-
 import { FYABackend } from '../../common/utils';
-import { getCurrentUser, auth } from '../../common/auth_utils';
-
+import { getCurrentUser, auth, logout } from '../../common/auth_utils';
+import { Cookies, withCookies } from '../../common/cookie_utils';
 import useMobileWidth from '../../common/useMobileWidth';
+import addBoxIcon from '../../assets/navBarIcons/add-box-icon.svg';
+import dashboardIcon from '../../assets/navBarIcons/admin-dashboard-icon.svg';
+import exportCSVIcon from '../../assets/navBarIcons/export-csv-icon.svg';
+import hamburgerIcon from '../../assets/navBarIcons/hamburger-icon.svg';
+import uploadCSVIcon from '../../assets/navBarIcons/upload-csv-icon.svg';
+import userProfileIcon from '../../assets/navBarIcons/user-profile-icon.svg';
+import logoutIcon from '../../assets/navBarIcons/logout-icon.svg';
+import aboutIcon from '../../assets/navBarIcons/about-icon.svg';
```

Comment:
🚫 **[eslint]** <[import/no-unresolved](https://github.com/import-js/eslint-plugin-import/blob/v2.26.0/docs/rules/no-unresolved.md)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Unable to resolve path to module '../../assets/navBarIcons/about-icon.svg'.